testsuite/run_sets/testsuite.yml

```diff
@@ -109,6 +109,8 @@
 - features/srv_sync_products.feature
 - features/allcli_reboot.feature
 - features/srv_notifications.feature
+- features/min_virt_kvm_guests.feature
+- features/min_virt_xen_guests.feature
```

Comment:
No. The `min_` prefix is reserved to normal minion. We must invent two new prefixes.

I suggest things like `minxen` and `minkvm`:
```
 - features/minkvm_guests.feature
￼ - features/minxen_guests.feature
```
We would also need to document the two new prefixes (there's a `.md` file for that).

Reply:
ok